pkg/runner/instance/process_instance.go

```diff
@@ -28,10 +29,20 @@ const (
 )
 
 var (
-	binary = "/proc/self/exe"
+	binary   = "/proc/self/exe"
+	policies = map[string]func(*processInstance) int{
+		"default": defaultPolicyfunc,
+	}
 )
 
+func defaultPolicyfunc(i *processInstance) int {
+	now := time.Now().Unix()
+	birthday := i.startTime.Unix() // happy birthday for LittleDrizzle. He write this line at the birthday (smddx)
```

Comment:
`Happy birthday to LittleDrizzle. He wrote this line on his birthday (smddx)`

Reply:
oh, my poor English